src/app/pages/system/general-settings/localization-form3/localization-form3.component.html

```diff
@@ -0,0 +1,19 @@
+<ix-form [formGroup]="form" (cancel)="cancelled()" [title]="formTitle" (formSubmit)="submit($event)">
+  <ix-input
+    label="Name"
+    name="name"
+    required="true"
+    tooltip="nameTooltip"
+    placeholder="namePlaceholder"
+    [ngModel]="name"
```

Comment:
Don't understand why ngModel is needed here, since we are using formControlName.

Reply:
Setting a default value (e.g., on edit forms), was not getting reflected on the UI. The form control would not pick up the value in the form group either. Have to use this `[ngModel]` to set default values for form controls.
On second that, it might not be needed to set the value here in the actual form component. But `ngModel` was needed to update the UI with the default value because otherwise, it wouldn't pick up the value. Still working on replacing this with just `FormControl` apis